### PR TITLE
Update with a new category - API Web Scanners

### PR DESCRIPTION
**Pull request by yigblst**
Added a free open source API web scanner - Cherrybomb

### CODE REVIEW

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 1]
```diff
@@ -139,6 +139,14 @@ this time:
     (CE)](https://www.contrastsecurity.com/contrast-community-edition) -
     Fully featured version for 1 app and up to 5 users (some Enterprise
     features disabled). Contrast CE supports Java and .NET only.
+    
+### API Web Scanners
+
+Tools which focus on web APIs.
+
+  - [Cherrybomb](https://github.com/blst-security/cherrybomb) - Cherrybomb is a CLI tool that helps you avoid undefined user behavior by validating your API specifications.
+      - It takes in an OAS file, runs a series of checks on it to make sure everything is on par with the OAS, and outputs a detailed table with any alerts found, guiding you to the exact problem and location to help you solve it quickly.
```
Comment (psiinon) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:148]: Looks a bit salesy to me ;)

**Review by kingthorin — COMMENTED** [review]
https://owasp.org/www-community/api_security_tools seems like a better place for this.

https://github.com/OWASP/www-community/blob/master/_data/api-tools.yml

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 2]
```diff
@@ -139,6 +139,12 @@ this time:
     (CE)](https://www.contrastsecurity.com/contrast-community-edition) -
     Fully featured version for 1 app and up to 5 users (some Enterprise
     features disabled). Contrast CE supports Java and .NET only.
+    
+### API Web Scanners
+
+Tools which focus on web APIs.
+
+  - [Cherrybomb](https://github.com/blst-security/cherrybomb) - Cherrybomb is a CLI tool that helps you avoid undefined user behavior by validating your API specifications.
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:147]:
```suggestion
For tools which are API specific please refer to the OWASP community [/api_security_tools](API Security Tools) page.
```

**Review by kingthorin — COMMENTED** [review]
Let's make this just refer to the other content now that cherrybomb has been added there.

**Review by kingthorin — APPROVED** [review]
Thanks

**Pull request merged**